tests/run_tests.sh

```diff
@@ -220,13 +220,13 @@ while getopts "h?c:d:e:f:i:k:l:m:n:op:rs:t:ux" opt; do
             show_help_and_exit 0
             ;;
         c )
-            TEST_CASES=${OPTARG}
+            TEST_CASES="${TEST_CASES} ${OPTARG}"
```

Comment:
Just a general question: does getopts just handle this (having multiple `-c` passed in) automagically? Or is there some other trick you're using here?

Reply:
I checked. getopts hands value one at a time. So the last -c (for instance) would take effect without this change.

Reply:
Sweet!